Sparkline chart changes (#222)
patternfly/training-scenarios: react-charts/sparkline-chart/step4.md

Add a label to the sparkline chart.

1) <strong>Make sure `App.js` is still open.</strong>

> 2) <strong>Locate the code that looks like this:</strong>

Reviewer: ```suggestion
2) <strong>Locate the code for the `chart-container` wrapper.</strong>
```

<pre class="file">
&lt;div className=&quot;chart-container&quot;&gt;
  ...
&lt;/div&gt;
</pre>

> 3) Add the following code inside the `<div>`

Reviewer: ```suggestion
3) Add the `ChartContainer` component outside of the wrapper identified in step 2.
```

(@dlabrecq can you confirm that this is right)

Reviewer: And then add a step for adding the Chart Label inside of the Chart Container. also explain why you need a chart container component

Reviewer: i think its safe to commit the suggestion above ^ @wojta

Author: >  Add the `ChartContainer` component outside of the wrapper identified in step 2.
outside?

Reviewer: Yep can confirm it sits below the identified section:
This is from the original Sparkline

<img width="408" alt="Screen Shot 2019-12-10 at 11 43 13 AM" src="https://user-images.githubusercontent.com/20118816/70549481-49501d00-1b42-11ea-85b1-619288417de9.png">
